<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qOrbit</class>
 <widget class="QMainWindow" name="qOrbit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qOrbit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSimpleSatelliteMap" name="mapWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPolarView" name="polarWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PassTable" name="tableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQObit">
    <property name="title">
     <string>qObit</string>
    </property>
   </widget>
   <addaction name="menuQObit"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPolarView</class>
   <extends>QWidget</extends>
   <header>qPolarView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSimpleSatelliteMap</class>
   <extends>QWidget</extends>
   <header>QSimpleSatelliteMap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PassTable</class>
   <extends>QTableWidget</extends>
   <header>PassTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
